.github/workflows/pytest.yml

```diff
@@ -69,6 +69,7 @@ jobs:
 
           - |
             pymc/tests/test_distributions.py
+          - pymc/tests/test_variational_inference.py
```

Comment:
Don't put it in this job, as it is already the longest one. You can simply remove the `--ignore` line from above. 

If the tests take very long you can put it in one of the next two instead, which are the fastest jobs currently